mon-pix/app/components/tutorials/card.js

```diff
@@ -37,6 +37,10 @@ export default class Card extends Component {
     return this.evaluationStatus !== buttonStatusTypes.unrecorded;
   }
 
+  get isTutorialButtonDisabled() {
```

Comment:
```suggestion
  get isEvaluationButtonDisabled() {
```
Puisqu'il y a 2 boutons différents, ça peut valoir le coup de les distinguer